consensus/istanbul/types.go

```diff
@@ -152,6 +152,18 @@ func (b *Subject) DecodeRLP(s *rlp.Stream) error {
 	return nil
 }
 
+func (a *Subject) Equal(b *Subject) bool {
+	if a == nil && b == nil {
```

Comment:
What about checking `a` output of the Equal?

Reply:
Logically nil object can call its method (Please check out `TestNilSubjectCmp` function). So, I think it seems required for consistency.

Reply:
Oh it's working. Thank you.